tests/utils.py

```diff
@@ -351,3 +355,32 @@ def _application_docker_container(
         container.reload()
     yield container
     container.remove(force=True)
+
+
+def find_application_pid(pid: int) -> int:
+    # Application might be run using "sh -c ...", we detect the case and return the "real" application pid
+    process = Process(pid)
+    if (
+        process.cmdline()[0].endswith("sh")
+        and process.cmdline()[1] == "-c"
+        and len(process.children(recursive=False)) == 1
+    ):
+        pid = process.children(recursive=False)[0].pid
+
+    return pid
+
+
+def assert_jvm_flags_equal(actual_jvm_flags: Optional[List], expected_jvm_flags: List) -> None:
+    """
+    When comparing JVM flags, we want to ignore the "value" field
+    as it might change in the CI due to ergonomic set flags
+    """
+    assert actual_jvm_flags is not None, f"{actual_jvm_flags} != {expected_jvm_flags}"
```

Comment:
Do you need the assert message? For this one and for the next one?

Reply:
yes, that way its clearer when it fails